lib/orion.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const Intercom = require('intercom-client')
+const envIs = require('101/env-is')
+const exists = require('101/exists')
+const Company = require('./company')
+
+/**
+ * Orion, the huntsman, in charge of helping us hunt down those big customers.
+ * Used for communication with intercom with very little mucking about configuration
+ * @class
+ * @author Ryan Kahn
+ */
+class Orion {
+  /**
+   * Creates a new Orion instance for reporting to intercom
+   */
+  constructor () {
+    this.company = new Company(this)
```

Comment:
just want to know the reasoning behind this. what else can inherit company? Looks like this logic should be moved inside Company because that object knows all about intercom.